gotime/go-time-101.md

```diff
@@ -428,13 +428,13 @@ Basically, they were causing a clash in security tokens just because they were l
 
 **Mat Ryer:** So you mentioned programmer laziness... It's an interesting point, but -- genuinely, a lot of teams rush; they build software extremely quickly, they're under a lot of pressure, and a lot of people, when you talk to them, they feel like that's how you build software. But there is a good argument to be had for taking your time, slow things down, spend a bit more time on it, and maybe we can start to avoid some of those things.
 
-**Roberto Clapis:** Yeah. One saying that we have is that "Optimists deploy soon and work fast, and they write post-mortems." \[laughter\] Pessimists write tests and Fuzz functions, and they kind of sleep safe at night.
+**Roberto Clapis:** \[50:41\] Yeah. One saying that we have is that "Optimists deploy soon and work fast, and they write post-mortems." \[laughter\] Pessimists write tests and Fuzz functions, and they kind of sleep safe at night.
```

Comment:
This new timestamp and the next one (51:39) don't follow the format of other timestamps, I'd recommend removing them since there's already one in the transcript shortly afterwards.